app/config/packages/framework.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+    secret: '%env(APP_SECRET)%'
+    #default_locale: en
```

Comment:
If this is commented, can't it be deleted?

Reply:
It was generated by symfony, but I think I can remove that line